### PR TITLE
IOS-11878 Show keypad when creating a folder

### PR DESCRIPTION
**Pull request by jlawton**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @jlawton, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXCreateFolderViewController.m [hunk 1]
```diff
@@ -114,6 +114,11 @@ - (void)viewWillAppear:(BOOL)animated
     self.navigationController.navigationBar.translucent = NO;
 }
 
+- (void)viewDidAppear:(BOOL)animated {
```
Comment (adempsey) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXCreateFolderViewController.m]: minor - conventionally, we put the trailing brace on a new line

**Comment by adempsey** [comment]
+2

**Comment by jlawton** [comment]
CLA signed

**Comment by boxcla** [comment]
Verified that @jlawton has just signed the CLA. Thanks, and we look forward to your contribution.

**Pull request merged**